packages/flutter/lib/src/cupertino/desktop_text_selection.dart

```diff
@@ -182,7 +182,6 @@ class _CupertinoDesktopTextSelectionControlsToolbarState extends State<_Cupertin
       }
 
       items.add(CupertinoDesktopTextSelectionToolbarButton.text(
-        context: context,
```

Comment:
I think that not passing this may break dark/light mode changes.  See the implementation here: https://master-api.flutter.dev/flutter/cupertino/CupertinoDesktopTextSelectionToolbarButton/CupertinoDesktopTextSelectionToolbarButton.text.html

Reply:
I see your code below where you refactored this a bit.  It should be alright, but please try this out in both light and dark mode and make sure it works in both.

I had some light/dark mode problems with the iOS toolbar and created an example app for that (in the description of PR https://github.com/flutter/flutter/pull/117234), you could probably do something similar for the desktop Mac toolbar.

Reply:
I think theme changes are OK, is this test enough or should I create the example app?

https://user-images.githubusercontent.com/12024080/224187032-94977d57-c87c-4768-9689-dad37dc37ba8.mov